content/patterns/multicloud-gitops-qat/mcg-qat-getting-started.adoc

```diff
@@ -0,0 +1,305 @@
+
+. Under *Git Config > Target Repo* copy the link to your fork and under *Git Config > Target Revision* write the name of your branch (Figure 3).
+
+. Click *Create* to create the pattern.
++
+.Create Pattern Form
+image::multicloud-gitops-qat/qat-create-pattern.png[Create pattern Multicloud GitOps with qat]
++
+
+
+Verify that the rest of Operators have been installed:
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *multicluster engine for Kubernetes* 
+* *Node Feature Discovery Operator* 
+* *Red Hat Openshift GitOps*
+* *Sail Operator*
+* *Intel Device Plugin Operator*
+
+Generate client and server certificates. User can refer to Istio official guide: https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#generate-client-and-server-certificates-and-keys
+
+After generating them add them as secrets to *Vault* manually:
+
+. Go to Vault service route. URL can be found:
+
+.. by running command:
++
+[source,terminal]
+----
+oc -n vault get route vault -ojsonpath='{.spec.host}'
+----
++
+.. in *Openshift Container Platform* web console under *Networking > Routes* for `vault` project.
+
+. Log into the Vault using root token. Root token can be found by executing command:
++
+[source,terminal]
+----
+oc -n imperative get secrets vaultkeys -ojsonpath='{.data.vault_data_json}' | base64 -d
+----
++
+
+. After login go to `secret` catalog and clik *Create secret* and fill all the fields manually (Figure 2):
+
+.. *Path for this secrets* should be look like this: `global/certs/httpbin.example.com.crt`, `global/certs/example.com.crt`, `global/certs/httpbin.example.com.key`.
+
+.. Under *Secret data* key is `secret` and in next field put its value.
+
+.. Click *Add* and then *Save*.
++
+.Create secret
+image::multicloud-gitops-qat/qat-secret-vault.png[Create secret in the vault]
++
+
+
+== Verification
+
+Go to the Hub ArgoCD and verify that all applications are synchronized. The URL can be found in *Openshift Container Platform* web console under *Networking > Routes* for the project `multicloud-gitops-qat-hub` or use command:
+
+[source,terminal]
+----
+oc -n multicloud-gitops-qat-hub get route hub-gitops-server -ojsonpath='{.spec.host}'
+----
+
+All applications should be `Healthy` and `Synced`:
+
+.ArgoCD panel with `qat-app`
+image::multicloud-gitops-qat/multicloud-gitops-argocd-qat.png[Multicloud GitOps Hub with `qat-app`]
+
+Check the logs of a pod `httpbin` to verify if it uses *{intel-qat}*. In the *OpenShift Container Platform* web console, navigate to *Workloads > Pods*. Change project to `httpbin` and open the *Logs* tab in the pod details. Go to container `istio-proxy` and check if there are no errors. 
```

Comment:
Check **[the httpbin pod logs]** to verify if it uses...

Reply:
done